news_flash/db_queries.py

```diff
@@ -0,0 +1,25 @@
+import psycopg2
+
+
+def get_latest_date_from_db():
+    conn = psycopg2.connect("dbname=anyway_news_flash user=postgres password=123321")
```

Comment:
you can use [context manager](http://initd.org/psycopg/docs/usage.html#with-statement) here so you won't have to remember to close your connections.

Reply:
Fixed